src/main/java/seedu/address/storage/csv/CsvAdaptedStudent.java

```diff
@@ -0,0 +1,65 @@
+package seedu.address.storage.csv;
+
+import java.util.stream.Collectors;
+
+import com.opencsv.bean.CsvBindByName;
+
+import seedu.address.model.student.Student;
+import seedu.address.model.tutorial.Tutorial;
+
+/**
+ * CSV-friendly version of {@link Student}.
+ */
+public class CsvAdaptedStudent {
+
+    private static final String TUTORIALS_SEPARATOR = ";"; // Used to separate multiple tutorials in a CSV field
+
+    @CsvBindByName
+    private final String name;
+    @CsvBindByName
+    private final String id;
+    @CsvBindByName
+    private final String phone;
+    @CsvBindByName
+    private final String email;
+    @CsvBindByName
+    private final String handle;
+    @CsvBindByName
+    private final String tutorials;
+
+    /**
+     * Converts a given {@code Student} into this class for CSV use.
+     */
+    public CsvAdaptedStudent(Student source) {
+        this.name = source.getName().fullName;
+        this.id = source.getStudentId().id;
+        this.phone = source.getPhone().value;
+        this.email = source.getEmail().value;
+        this.handle = source.getHandle().handle;
+        this.tutorials = String.join(TUTORIALS_SEPARATOR,
+                        source.getTutorials().stream().map(Tutorial::toString).collect(Collectors.toList()));
```

Comment:
Should the processing logic for tutorials be extracted into a helper method? It seems like the constructor is handling inline transformation, which might affect readability and Single Level of Abstraction Principle. An extracted method could improve clarity and reusability. 

```suggestion
        this.tutorials = formatTutorials(source.getTutorials());
}

private static String formatTutorials(List<Tutorial> tutorials) {
    return tutorials.stream().map(Tutorial::toString).collect(Collectors.joining(TUTORIALS_SEPARATOR));
}
```